src/lib/components/Header.svelte

```diff
@@ -1,5 +1,12 @@
 <script>
 	import logo from '$lib/assets/brand.svg';
+	import Hamburger from './Hamburger.svelte';
 </script>
 
-<div class="flex items-center justify-center"><img src={logo} alt="rack em up" class="my-2" /></div>
+<div class="grid grid-cols-4 place-items-center max-w-[600px] m-auto relative">
+	<div class="col-span-1"></div>
+	<img src={logo} alt="rack em up" class="my-2 col-start-2 col-end-4" />
+	<div class="col-start-4">
```

Comment:
```suggestion
	<img src={logo} alt="rack em up" class="my-2 col-span-4" />
	<div>
```

Reply:
This suggestion doesn't seem to work.  col-span-4 on the img pushes the cheeseburger onto a new line. 

Additionally it wouldn't work with the intent as there is a place holder div currently in the first grid slot. That will be used for the back arrow when it's implemented. 

<div class="grid grid-cols-4 place-items-center max-w-[600px] m-auto relative">
	<div class="col-span-1"></div>
	<img src={logo} alt="rack em up" class="my-2 col-start-2 col-end-4" />
	<div class="col-start-4">
		<Hamburger />
	</div>
</div>

Reply:
I tried to send the code but it didn't let me